packages/picasso.js/src/web/text-manipulation/font-size-to-height.js

```diff
@@ -0,0 +1,25 @@
+const BASE = 24;
+const PAD = 4;
+const BUMP = 1e-12;
+const DEFAULT_HEIGHT = 16;
+const TEXT_REGEX = /^\s*([0-9]+|[0-9]+\.{1}[0-9]+)[px]{2}\s*$/i;
```

Comment:
simplify the regex to `/^\s*\d(\.\d+)?px\s*/`

Reply:
Doesn't work for multiple number, ex: 90px. But will fix n update.

Reply:
Also regarding case, I've tried to follow the browser implementation for SVG text element. Where in FF, chrome and safari, PX and px is allowed.